src/lib/wallet/IndyWallet.ts

```diff
@@ -76,6 +76,52 @@ export class IndyWallet implements Wallet {
     return this.indy.issuerCreateCredentialOffer(this.wh, credDefId);
   }
 
+  public async createCredentialRequest(
+    proverDid: string,
+    offer: CredOffer,
+    credDef: CredDef,
+    masterSecretName: string
+  ): Promise<[CredReq, CredReqMetadata]> {
+    if (!this.wh) {
+      throw Error('Wallet has not been initialized yet');
+    }
+    // TODO save `masterSecret` during wallet init and just use it in `proverCreateCredentialReq`
+    const masterSecretId = await this.indy.proverCreateMasterSecret(this.wh, masterSecretName);
+    return this.indy.proverCreateCredentialReq(this.wh, proverDid, offer, credDef, masterSecretId);
+  }
```

Comment:
Does this mean that credentials cannot be linked at the moment as all credentials use a new `masterSecret`?

Reply:
I didn't read your comment in the PR.